docs/themes/thxvscode/layouts/partials/apiNav.html

```diff
@@ -80,9 +84,15 @@
     </ul>
     <hr/>
     <ul class="nav" id="secondary-nav">
-        <li {{ if $in_api_section }} class="active" {{ end }}>
-            <a class="docs-home" href="/docs/apis/" {{ if $in_api_section }}
-                aria-label="Current Page: API Overview" {{ end }}>API Overview</a>
+        <li>
+            <select name="API version" id="api-version" onchange="window.location.href = this.value;">
+                {{ $selectedApiVersion := trim $current.RelPermalink "/" }}
+                {{ $selectedApiVersion = index (split $selectedApiVersion "/") 2 }}
+                {{ range $version, $path := $.Site.Data.versions.apiDocContentPath }}
+                    {{ $isCurrentVersion := strings.Contains $version $selectedApiVersion }}
```

Comment:
Can this not be an exact equality check? Curious if `contains` is necessary. If so, it would be good to add a comment explaining why.

Reply:
If we won't have version number like "1.0.1" then we can use `contains`. If we will then we need the exact match.

Reply:
Oh, I see. For the purpose of this system, when we release package versions `2.0.1`, we'll publish their artifacts as `2.0.0-latest`, rather than publishing separate artifacts for each version.

Also note that we will eventually publish versions like `2.1.0`, which would fail this check.

Reply:
But this highlights the fact that our versioned artifact naming is a bit weird. It might make more sense for us to name our versions here something like `1.x` and `2.x` instead of `1.0` and `2.0`. Or even just `v1` and `v2`. We should discuss this during the next docs sync meeting.

Reply:
Marking the following discussion here.

Reply:
Discussed in meeting. Name to v1, v2 now